### PR TITLE
Milestone 1: Setup project

### PR DESCRIPTION
**Pull request by medaminedev66**
- Create an npm project with webpack.
- Write plain HTML markup with minimum styling

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+
+module.exports = {
+  entry: './src/index.js',
+  devServer: {
+    contentBase: './dist',
```
Comment (Davidosky007) [inline webpack.config.js]: * Kindly fix this error i get when i run `npm start` it says invalid options object 
* Hint 💭 you can see the option contentBase  is not a valid devServer option , you can use any valid option stated bellow ⬇️ 
Screenshot (175)

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
## Hi @medaminedev66 👋, 

Good job so far! 💪 
* The repo looks professional.✔️
* Git/GitHub flow best practices have been followed✔️
*  There are no linter errors✔️
* You have descriptive PR✔️
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: webpack.config.js (hunk 1 above)
Comment (medaminedev66) [reply webpack.config.js]: Hi @Davidosky007, thank you for reviewing my project. I thought the webpack server is working. Ok, I will try to fix it.

Comment (rloterh) [reply webpack.config.js]: This issue is well resolved! ✔️

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Amine, 

Your project is complete! Great job for making the necessary changes recommended by the previous reviewer. There is nothing else to say other than... it's time to merge it 🍾 :shipit: .
Congratulations! 🎉 🎉 🥂

<br>
Cheers and Happy coding!👏 💻 🍷 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 1 more limited review per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by medaminedev66** [comment]
Thank you for taking the time to review my project.

**Pull request merged**